packages/api/src/domain/base-domain.ts

```diff
@@ -2,9 +2,12 @@ export interface BaseDomainCreate {
   id: string;
 }
 
-export interface BaseDomain extends BaseDomainCreate {
+export interface BaseDomainNoId {
   createdAt: Date;
   updatedAt: Date;
+}
+
+export interface BaseDomain extends BaseDomainCreate, BaseDomainNoId {
   eTag: string;
 }
```

Comment:
This is to allow a table with no `ID`, it was used here to have a composite key column.

Reason for the composite key:
- this is a "log" table, not going to be referenced in the actual product;
- needed a multi-index column so we can search by EC run ID + patient ID.